examples/sftp_get.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from pylibsshext.sftp import SFTP
+from pylibsshext.session import Session
+
+
+HOST = '192.168.122.9'
+USER = 'akahat'
```

Comment:
Can you change these values to something generic?

Reply:
okay. Cool.